scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala

```diff
@@ -167,6 +169,16 @@ final class ScioContextOps(private val self: ScioContext) extends AnyVal {
   ): SCollection[T] =
     self.read(AvroTypedIO[T](path))(AvroTypedIO.ReadParam(suffix))
 
+  /**
+   * Read avro data from `path` as `GenericRecord` and convert to `T` via the implicitly-available
+   * `magnolify.avro.AvroType[T]`
+   */
+  def typedAvroFileMagnolify[T: MagnolifyAvroType: Coder](
```

Comment:
IMHO this was a mistake to name that `typedAvro...`. This should have been `avroTyped...`.
This improves developer experience when APIs are grouped by prefix

Reply:
Agree, I've always found the `typed` prefix annoying. We could scalafix it I guess?

Reply:
yes, the question is if we want that change for 0.15? We can discuss this later anyway. I'd better get this change in another PR